developer_manual/design/popovermenu.rst

```diff
@@ -0,0 +1,68 @@
+.. sectionauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+.. codeauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+..  _popovermenu:
+
+===============
+Popover Menu
+===============
+
+What is a popover menu
+=======================
+
+This is a quick menu that open on click. We're usually using the three dots icons on nextcloud.
```

Comment:
open**s**

»For menus, we use the three-dot-icon.« (also »Nextcloud needs to have a capital N ;)